soccer/planning/planner/CollectPlanner.hpp

```diff
@@ -1,8 +1,8 @@
 #pragma once
 
-#include "DirectTargetPathPlanner.hpp"
-#include "RRTPlanner.hpp"
-#include "SingleRobotPathPlanner.hpp"
+#include "planning/Instant.hpp"
```

Comment:
This is almost line-for-line the same as the original collect planner. There should be no major functional changes.